### DISCUSSION TITLE
Endpoint optimize problem

### DISCUSSION THREAD

**Discussion by liyunshan123** [apache/skywalking#10711] [Q&A]
Skywalking version: 9.3.0

![image](https://user-images.githubusercontent.com/400354/233605688-14bdb726-a6c1-4a70-8bd9-258877d4673d.png)
Our project use Hystrix as service circuit breaker，but there are many more endpoints starting with Hystrix/. 
We tried to block these endpoints through Innore trace plugin, but there was another issue where we couldn't find trace chains below Hystrix/* in trace pages.
![image](https://user-images.githubusercontent.com/400354/233607576-f7fd7134-6b25-482b-acd5-439c001016d1.png)

We don't want to display endpoints begin with Hystrix, but need trace chains display the endpoints begin with Hystrix. How to resolve the problem.

**Comment by wu-sheng** [comment]
First of all, this is not an issue at all. You seem to have an unrealistic understanding of your running applications.
___
The Hystrix is used in your processing chain due to your dev team choices, if you skip them, the traces broke as you had seen.
APM system is showing what is really happening in your applications, visualizes them, and runs statistics for them.

**Comment by wu-sheng** [comment]
Then, the endpoint page only shows very few endpoints from all. You should always search for which you are interested.
